static/styles/left_sidebar.css

```diff
@@ -199,18 +247,62 @@ li.show-more-topics {
     width: 100%;
 }
 
-#private-container {
-    max-height: 210px;
+#private_messages_section {
+    margin-bottom: 8px;
+    margin-right: 12px;
+}
+
+#private_messages_list {
+    padding-top: 2px;
+    padding-bottom: 2px;
+}
 
-    /* Match the opacity for global-filters icons. */
-    span.fa-group {
-        opacity: 0.7;
+.active_private_messages_list {
+    background-color: hsl(202, 56%, 91%);
+    border-radius: 4px;
+}
+
+.private-messages-section-header {
+    a {
+        display: inline;
+        color: inherit;
+        text-decoration: none;
+        outline: none;
+        border: none;
+    }
+
+    .unread_count {
+        margin-right: 42px;
+    }
+}
+
+#default_private_messages_view_header {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 9px;
+    cursor: pointer;
+    margin-top: 3px;
+}
+
+#more_private_messages_view_header {
+    display: flex;
+    justify-content: space-between;
+
+    #more-private-conversations-sidebar-title {
+        margin: 0;
+        padding-left: 13px;
+        color: inherit;
+    }
+
+    .active-more-private-conversations-sidebar-title {
+        font-weight: bold;
     }
 }
 
 :not(.active-sub-filter) {
     &.top_left_row:hover,
     &.bottom_left_row:hover,
+    &.expanded_private_message:hover,
```

Comment:
Ahh, I think we can delete this line and should instead add the `bottom_left_row` CSS class to the element; see https://github.com/zulip/zulip/pull/21726/commits/1a7646b4fdcaba1e377e2a9b9f6cc479722baece#r944038491

Reply:
Updated the PR, Thanks!